src/set_theory/zfc/basic.lean

```diff
@@ -874,6 +876,8 @@ def to_Set (B : Class.{u}) (A : Class.{u}) : Prop := ∃ x, ↑x = A ∧ B x
 protected def mem (A B : Class.{u}) : Prop := to_Set.{u} B A
 instance : has_mem Class Class := ⟨Class.mem⟩
 
+@[simp] theorem mem_empty (x : Class.{u}) : x ∉ (∅ : Class.{u}) := λ ⟨_, _, h⟩, h
```

Comment:
```suggestion
@[simp] theorem not_mem_empty (x : Class.{u}) : x ∉ (∅ : Class.{u}) := λ ⟨_, _, h⟩, h
```

Reply:
This matches `pSet.mem_empty` and `Set.mem_empty` but not `set.not_mem_empty`. I'll apply this change, then homogenize the naming scheme in another PR.